Documentation/articles/getting_started/toc.yml

```diff
@@ -4,11 +4,15 @@
     href: 1_creating_a_new_project_vs.md
   - name: Visual Studio for Mac
     href: 1_creating_a_new_project_vsm.md
-  - name: DotNet core CLI
+  - name: .Net core CLI

```

Comment:
.NET Core*

Reply:
@SimonDarksideJ  ".NET Core" not ".Net core".